cms/envs/common.py

```diff
@@ -228,14 +228,9 @@
 for namespace, template_dirs in lms.envs.common.MAKO_TEMPLATES.iteritems():
     MAKO_TEMPLATES['lms.' + namespace] = template_dirs
 
+# Django templating
 TEMPLATE_DIRS = MAKO_TEMPLATES['main']
```

Comment:
Do we still need these TEMPLATE_* settings? The Django 1.8 migration docs suggest moving this into the TEMPLATES setting and getting rid of all the TEMPLATE_*.